pygmt/modules.py

```diff
@@ -12,6 +12,26 @@
 )
 from .exceptions import GMTInvalidInput
 
+SPECIAL_PARAMS = {
+    "FONT": [
+        "FONT_ANNOT_PRIMARY",
+        "FONT_ANNOT_SECONDARY",
+        "FONT_HEADING",
+        "FONT_LABEL",
+        "FONT_TAG",
+        "FONT_TITLE",
+    ],
+    "FONT_ANNOT": ["FONT_ANNOT_PRIMARY", "FONT_ANNOT_SECONDARY"],
+    "FORMAT_TIME_MAP": ["FORMAT_TIME_PRIMARY_MAP", "FORMAT_TIME_SECONDARY_MAP"],
+    "MAP_ANNOT_OFFSET": ["MAP_ANNOT_OFFSET_PRIMARY", "MAP_ANNOT_OFFSET_SECONDARY"],
+    "MAP_GRID_CROSS_SIZE": [
+        "MAP_GRID_CROSS_SIZE_PRIMARY",
+        "MAP_GRID_CROSS_SIZE_SECONDARY",
+    ],
+    "MAP_TICK_LENGTH": ["MAP_TICK_LENGTH_PRIMARY", "MAP_TICK_LENGTH_SECONDARY"],
+    "MAP_TICK_PEN": ["MAP_TICK_PEN_PRIMARY", "MAP_TICK_PEN_SECONDARY"],
+}
```

Comment:
Also add **MAP_DEFAULT_PEN**:

```
MAP_DEFAULT_PEN
Sets the default of all pens related to -W options. Prepend + to overrule the color of the parameters
MAP_GRID_PEN_PRIMARY, MAP_GRID_PEN_SECONDARY, MAP_FRAME_PEN, MAP_TICK_PEN_PRIMARY, 
and MAP_TICK_PEN_SECONDARY by the color of MAP_DEFAULT_PEN [default,black].
```

Reply:
This one has a default setting `[default,black]` no?

Reply:
You're right. This parameter is irrelevant.